backend/copr-backend.spec

```diff
@@ -185,15 +188,20 @@ cp -a docs/build/html %{buildroot}%{_pkgdocdir}/
 install -d %{buildroot}%{_mandir}/man1
 install -p -m 644 copr-backend-resultdir-cleaner.1 %{buildroot}/%{_mandir}/man1/
 
+%if 0%{?fedora} && 0%{?fedora} > 41
+install -m0644 -D conf/copr-backend.sysusers.conf %{buildroot}%{_sysusersdir}/copr-backend.conf
+%endif
 
 %check
 ./run_tests.sh -vv --no-cov
 
+%if 0%{?fedora} && 0%{?fedora} < 42
```

Comment:
So this line will omit the user creation also in openEuler and RHEL?
I think we still need the code block in this scenario

Reply:
The existing Copr instances (four driven by us, one driven by you) - these all prepare the users/groups manually, so it is not an issue.  If this causes any problems to real deployments, we may consider partial revert.